CHANGELOG.md

```diff
@@ -1,26 +0,0 @@
-# Changelog
```

Comment:
Rather than removing this, I would recommend a deprecation notice at the top. See the reckoner repo for a similar situation.